Dockerfile

```diff
@@ -1,23 +1,29 @@
-FROM python:3.6.5
+FROM python:3.6.10-slim
+
+ADD ./source.list /etc/apt/source.list
+
+RUN apt-get update &&\
+    apt-get install --yes git
 
 # pass with --build-arg SETUPTOOLS_SCM_PRETEND_VERSION=VERSION; this is needed by setuptools_scm
-ARG SETUPTOOLS_SCM_PRETEND_VERSION
+ARG SETUPTOOLS_SCM_PRETEND_VERSION=dev
```

Comment:
Why are you using "dev" as version here?
Can't we obtain a better value?

Reply:
This is a default just in case user doesn't provide version while building (check `build.sh` script)

Reply:
OK =)